src/app/transaction-deny/components/TransactionDenyInput.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+import { useState } from "react";
+
+import { Button } from "@/components/ui/Button";
+import { Card } from "@/components/ui/card";
+import { putObjectsAsZip } from "@/data/object";
+import { updateTransaction } from "@/data/transaction";
+
+interface Props {
+  id: string;
+  setSendingStatus: (val: string) => void;
+}
+
+const TransactionDenyInput = ({ id, setSendingStatus }: Props) => {
+  const [files, setFiles] = useState<File[]>([]);
+  const [details, setDetails] = useState("");
+  const [detailNotProvided, setDetailNotProvided] = useState("");
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (!event.target.files) return;
+    setFiles([...files, ...Array.from(event.target.files)]);
+    event.target.value = "";
+  };
+
+  const validateInput = (): boolean => {
+    if (!details) {
+      setDetailNotProvided("Please provide some details.");
+      return false;
+    }
+    return true;
+  };
+
+  const handleSubmitClick = async () => {
+    if (!validateInput()) return;
+    setSendingStatus("sending");
+
+    try {
+      const uploadFolder = "report_envidence";
```

Comment:
```suggestion
      const uploadFolder = "report_evidence";
```

If this folder name will be used on other part of the app, please make it a global constant variable to prevent spelling error